concepts/05 Widgets/Scheduler/030 Appointments/015 Appointment Types/030 Recurring Appointments.md

```diff
@@ -1,40 +1,32 @@
-A recurring appointment is an appointment that repeats after a specified period of time. 
+A recurring appointment is an appointment that repeats at a certain interval. 
 
 ![Scheduler Recurring Appointment](/images/UiWidgets/Scheduler_RecurrentAppointment.png)
 
-An object defining a recurring appointment should contain the [recurrenceRule](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/recurrenceRule.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule') field that follows the <a href="https://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification.
+To make an appointment recurring, use the [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule) option (it follows the <a href="https://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification). You can also exclude certain appointments from a series. Use the [recurrenceException](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceException) option to specify their start date and time.
+
+The appointment below occurs daily since February 20, 2021 - except February 22 and 23, 2021:
```

Comment:
A comma seems more appropriate here that a dash.

Reply:
Removed the dash, but didn't add a comma because the sentence doesn't deliver the main idea if "except February 22 and 23, 2021" is omitted.